factorgraph/factorgraph.py

```diff
@@ -263,7 +263,7 @@ def add_factor(self, factor):
         self._factors += [factor]
 
     def joint(self, x):
-        '''
+        r'''
```

Comment:
Tiny nit: could you please remove the `r`? I think it'd be best to remain consistent with the other function comments.

Reply:
Sure. The reason I added it was because the comment contains `\product`, and the \p is interpreted in the string as a control character, so a warning is emitted in tests:

```
py-factorgraph % pytest 
================================================================ test session starts =================================================================
platform darwin -- Python 3.9.1, pytest-6.2.2, py-1.10.0, pluggy-0.12.0
rootdir: /Users/yan/Documents/GitHub/py-factorgraph
plugins: cov-2.11.1, xdist-2.2.1, forked-1.3.0
collected 2 items                                                                                                                                    

tests/e2e_test.py ..                                                                                                                           [100%]

================================================================== warnings summary ==================================================================
factorgraph/factorgraph.py:266
  /Users/yan/Documents/GitHub/py-factorgraph/factorgraph/factorgraph.py:266: DeprecationWarning: invalid escape sequence \p
    '''

-- Docs: https://docs.pytest.org/en/stable/warnings.html
============================================================ 2 passed, 1 warning in 0.26s ============================================================
```

An alternative would be to write `\\product` - would that be preferable?

Reply:
Oh shoot, great catch! No I agree with your modification, then, `r'''` is great!